### ISSUE TITLE
use with rubber ducky?

### ISSUE THREAD

**Issue by rocco7657** [t3l3machus/hoaxshell#56]
is it possible to make hoaxshell generate the same payload every time so you can use a rubber ducky

**Comment by k7tz** [comment]
Yes, you need to use the same authorization code, i have code especially for the usb rubber duck with the hoaxshell on my GitHub page

**Comment by WIWI84** [comment]
> Yes, you need to use the same authorization code, i have a tutorial for the usb rubber duck with the hoaxshell on my GitHub page
